0011-interledger-payment-request/0011-interledger-payment-request.md

```diff
@@ -0,0 +1,135 @@
+
+### Scope
+
+This document defines a suggested format for representing payment requests and an approach for receiver-generated conditions. Protocols built on top of Interledger MAY use the format provided here, though they can use any format they choose instead.
+
+While this document also provides a recommendation for generating transfer conditions, conditions are often opaque to all parties aside from the receiver. Receivers MAY use any condition they choose, however this specification includes best practices that receivers SHOULD take into consideration.
+
+This document does **not** specify how payment requests are communicated from a recipient to a sender.
+
+### Model of Operation
+
+Interledger Payment Requests are intended to be used in higher-level protocols as follows:
+
+1. Recipient creates the payment request and generates the condition using one of algorithms outlined in [Condition Generation](#condition-generation).
+2. Recipient communicates the payment request to the sender.
+3. Sender generates an ILP Packet from the payment request.
+4. Sender prepares a local ledger transfer to a connector, held pending the execution condition given in the payment request and with the ILP Packet attached.
+5. Connectors forward the payment until it reaches the recipient.
+6. Recipient receives a notification of a held transfer with an ILP Packet attached.
+7. Recipient checks the transfer amount against the ILP Packet and checks the payment request has not expired.
+8. Recipient regenerates the condition and fulfillment from the ILP Packet attached to the incoming transfer, using the same algorithm as before, and fulfills the transfer's condition. If the condition generated does not match the one in the transfer it means that the ILP Packet has been tampered with and the transfer goes unfulfilled.
+
+## Payment Request Specification
+
+### JSON
+
+```json
+{
+    "address": "ilpdemo.red.bob.b9c4ceba-51e4-4a80-b1a7-2972383e98af",
+    "amount": "10.25",
+    "expires_at": "2016-08-16T12:00:00Z",
+    "additional_routing_info": {
+        "rate_curves": {
+            "bitcoin.": [[0,0],[1,400]]
+        } 
+    },
+    "additional_headers": "asdf98zxcvlknannasdpfi09qwoijasdfk09xcv009as7zxcv"
+}
+```
+
+| Field | Type | Short Description |
+|:--|:--|:--|
+| `address` | ILP Address | Request-specific address |
+| `amount` | Decimal String | Amount requested by the recipient |
+| `condition` | Crypto Condition | Execution condition for the payment |
+| `expires_at` | [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) Timestamp | Expiry of the request |
+| `additional_routing_info` | Object | Details used by the sender's client to route the payment |
+| `additional_headers` | Base64 String | Additional headers for the ILP packet |
+
+#### address 
```

Comment:
Correct. If we're using request-specific addresses it's not really an "account" address anymore. It's... just an address for that request